.travis.yml

```diff
@@ -14,6 +14,7 @@ env:
   - secure: rXw5vCn+dOVJvMAdnvW0j+t1hSY1I5pvk7vyXgLy4b1XKX/dJ8AJCOfeKGTcIlpKPYj4CipQn1g2yqVznBRhRNylt4rdCRSCIgZE0zUXSw9nDJ4OTz9mGW0gSEvHvCntlCcnG3UR8sO4WBJxysycpC37x8XyoGeRj5yEhahNqkh1U5R7ZVDlJKfo+el+FbFe2poOgffCrBahAYEMy5vtTVmnb+vwAejq2/1nS20EwxieVP3maroMalWkQKLkku9s10SpgQvK9FmCr02FWew7NnouzSROZr+t3mL0xv9CHN/6/83tSd+lDJFfRitHTImcQfIbvKoNpnPVqRuLemB7Jo6BWU/NlKcR0/s0Ef1ewT3ONrSrk41/8pRB0Vm24gc1EAL0Uzi9vl4PDQsEf4vxEt5AAdL2aJemVSRWORs5Wvm1qdB6hgYTzedZV/+pXFQrot8Fw1SzlHhb3ASm8/qTJIHNPCXUDHQgbNN1MconZoTDDjeythe2GyfnfCLMq0aMDGp1nes4CA/8miLkubm5iD9VgXiWAVPXxxWifwx59uph12OooFMWpNQHH7MoztflAsMTQ/bbHRVibp0J5FlGYxncOjCNJwgtShpEA2mICGB2IG75cu4asmDprABK7MP+qEe01hpL2dzGuxBY8/ckIwcbsEOQItPiRMmr7KXXz8c=
   - SENTRY_ORG=lesswrong
   - secure: aqPakii7j/8zWAgEwGAwgQfYFEYrA6WJeS2iZBJ8vOpzHk5jGuOKDsCKtFvTQ4VqE9MdXyCjnYalEseS9t5VwwFskf/e7NqhVNld0whPQEBqGKuR9PIzXwlvKeQZ2i34wVW6nCh3H+nDWjXLpSluyj7lN23Yog9Lrtpi/bs/dYualw/lseQH7ZbNwOFIdHpbKgZS8jJ+0LAc8lAlT/UldJGUgL72r2S4wtCb+9wMf5uUoA8IZrdn08lm1owmg8bzfxAwWaJSrV2xnOG2mHUC3IPwrzgGD0IOxhwB/hA7wLYBx982zHRCAh3kKd3kjqgIAAdvt1hMm86eiPM4i4ofa+O0O5/kxiBKS9fRxrFSsLxHvg5kbsgjsT0aNiisk9yqy22VCqGa8JilnjqWXiB0YpJIvE99Mba8zNfLqS+P2R+GjOOqo5+1Jgo1HCZonJABY4IuRw0FTI3EE2BOVfjfECJa7eu96udfKN323NVconsWSCejDgoF8kt0W5c6qNF4QSJSHWYQR2kDrfB8JxAeHoZK7GKGJKuIvAMiIp9OZc+UEV5lLy3Cb+f1Lpjq9Fjamvsa/SsawC5Ix4YQfB19oin86mMlJKrA1GBSPYSOKxQxjQ/2gfx9kYTf/OEQJeEtwGVD8v/spAVOKcjaOKKL2hVFrrnZA2p3nC+VkjzbNmE=
+  - secure: EpMeTiZqRWHXtH8/QTGMsVWcwhIS45zC9p3xMqO5DqZhv0jL4w3A1YzT50+AqKa8ane/ufNnm4Q/vctOngyObM8Sv0OlbHgzrtYROygyZnoGhfC4bZweuvtqX+c0NPYhyxlwZp6NxL2BQx41I0VaZemuvsiOnBXaZ5koO3kZFa2OiQmfHLCzLddOessBLxQvyrIqIw1imZNRLrJ+Pas/1LvFZa7e7zBoBwbimWJRuOJCILe7uwMTHFJqfOeK9uSVfg2lihAqMk3UE3lJR07LyyMFBpDNrrra1OjFKwvFuI2mfQE+s2NdgVFYoBpV3U1phRJ/79FD65mRIPabCZW8QP4VImdb/5GxojPMcY2ebTBrGX0FlDzbNhb+PZgovFiddMmNAOF7qrVzkaTl3OVBjv9B0loJ0nsVRCbg6E5AX47mIy3Ei8hqnVr35pmYkwaA4u+Gpry3RulJmdL5ddNT+vSv0gXuU2FVS6qdSMAyXGgH8mXdAF+ygpqw5bSBI+NDJ0JztA0RfSldI2cUl+62mkHpjvbBxc869mEZsf8oxqq1ONqoq1A8plefyhKMZS+9pYfbX4TCAQk4/xKJEHUuj9sZGqHeSF46zInRuQK/rKq+/iA7I/iUXK3G+UcFCr6nCb9X9GRQpsNL0s/BMUv/354ydChnnbk5LUoA2BtTZxw=
```

Comment:
Should there also be a line deleted here, in addition to the line added?

Reply:
Yep, but I do not know which one, and don't have a way to figure out which one.